public_html/wp-content/plugins/wc-post-types/inc/favorite-schedule-shortcode.php

```diff
@@ -181,6 +183,12 @@ function preprocess_schedule_attributes( $attr ) {
 		$attr['session_link'] = 'permalink';
 	}
 
+	// Sites without the `content_blocks` skip flag use blocks, so we need to prevent these from using `anchor`.
+	if ( ! wcorg_skip_feature( 'content_blocks' ) ) {
+		$attr['speaker_link'] = 'anchor' !== $attr['speaker_link'] ? $attr['speaker_link'] : 'permalink';
+		$attr['session_link'] = 'anchor' !== $attr['session_link'] ? $attr['session_link'] : 'permalink';
```

Comment:
```suggestion
		$attr['session_link'] = 'anchor' === $attr['session_link'] ? 'permalink' : $attr['session_link'];
```

`MAYBE` - This feels like a more straight-forward way to write it to me; if there's a negation there then I have to do an extra mental cycle to reverse it before the statement clicks for me. 

Same for the one above it.

Reply:
I disagree- I think about it a different way: we want to fall through to the selected link type, unless it's anchor.

I've left this, but updated the two comments above.

Reply:
WFM :shipit: